pytorch_pfn_extras/training/extension.py

```diff
@@ -222,3 +224,32 @@ def decorator(ext: 'ExtensionLike') -> 'ExtensionLike':
 
 def _as_extension(ext: 'ExtensionLike') -> Extension:
     return ext if isinstance(ext, Extension) else _WrappedExtension(ext)
+
+
+class ExtensionEntry:
```

Comment:
Please add docstring and add to Sphinx docs.

Reply:
Added docstring. PTAL!

Reply:
Could you add to Sphinx docs as well?

Reply:
Fixed.